articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -82,9 +84,11 @@ Once the resources are created, add the following environment variables. These w
 
 ## Register event handler to reload data from App Configuration
 
+Ensuring that users get their most up to date configurations is nearly impossible as a configuration can change at any moment (even after a request is sent out). To ensure users get their most up to date configurations we integrate a `synchronization token`. This `Synchronization token` guarantees users receive all changes at least up to the point which the notification was triggered. There still is no guarantee that changes occuring after a push notification was triggered will be reflected in the response.
```

Comment:
There is a similar discussion after the code snippet. This appears redundant. Suggest removing.